### PR TITLE
Added formik with Yup and implemented in login

### PR DESCRIPTION
**Pull request by ketanp-techuz**

### CODE REVIEW

**Comment by ashishtechuz** [comment]
Please fix eslint warnings to let deployment happen. Any warning while generating a build will fail the deployment.

**Comment by ketanp-techuz** [comment]
> Please fix eslint warnings to let deployment happen. Any warning while generating a build will fail the deployment.

Fixed, warning because of it's not sync with master.

File: src/pages/Login/Login.jsx [hunk 1]
```diff
@@ -1,44 +1,59 @@
...
+  });
+
+  const formik = useFormik({
+    initialValues: {
+      email: "",
+      password: "",
+    },
+    validationSchema: validationSchema,
+    onSubmit: (values) => {
+      console.log("values", values);
```
Comment (ashishtechuz) [inline src/pages/Login/Login.jsx:30]: Please remove console log after testing.

**Review by ashishtechuz — CHANGES_REQUESTED** [review]

**Review by ashishtechuz — APPROVED** [review]

**Comment by ketanp-techuz** [comment]
[Added formik with Yup and implemented in login](https://trello.com/c/qvvZsanP/6-added-formik-with-yup-and-implemented-in-login)

**Pull request merged**